RedditSharp/Things/VotableThing.cs

```diff
@@ -158,5 +169,41 @@ public void ClearVote()
             var response = request.GetResponse();
             var data = WebAgent.GetResponseString(response.GetResponseStream());
         }
+        
+        public void Report(ReportType reportType, string otherReason = null)
+        {
+            var request = WebAgent.CreatePost(ReportUrl);
+            var stream = request.GetRequestStream();
+
+            string reportReason;
+            switch (reportType)
+            {
+                case ReportType.Spam: 
+                    reportReason = "spam"; break;
+                case ReportType.VoteManipulation: 
+                    reportReason = "vote minipulation"; break;
```

Comment:
Still not how you spell manipulation. Also, you can just add commits to the earlier pull request to update it.